content/custom-authentication.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Custom Authentication Integration
+---
+
+* TOC
+{:toc}
+
+## Create a new user / Sign in
+
+<p class='info'><strong>Authentication types</strong>: Public OAuth Client ID</p>
+
+    POST https://everydayhero-staging.com/api/v2/authentication/sign_up
```

Comment:
"everydayhero-staging"
Please no?
Either production examples or Sandbox. 
We're trying really hard to keep staging away from end users in any form